Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1116,4 +1117,88 @@ The purpose of the security control is to prevent the creation of Service Princi
 
 <br />
 
+___ 
+
+## Azure_Subscription_AuthZ_Dont_Grant_SPNs_Privileged_Roles 
+
+### Display Name 
+ Service Principals must follow Least privilege principle for role assignments to Subscriptions 
+
+### Rationale 
+SPNs have a single credential and most scenarios that use them cannot support multi-factor authentication. Also, SPNs and Managed Identities can't be granted Just-In-Time access. As a result, adding SPNs to a Subscription with privileged roles is risky.  
+
+### Control Spec 
+
+> **Passed:** 
+> Critical roles (Owner, Contributor or User Access Administrator) are not assigned to SPNs at subscription level.
+> 
+> **Failed:** 
+> Critical roles (Owner, Contributor or User Access Administrator) are assigned to SPNs at subscription level.
+>
+
+### Recommendation 
+
+- If these SPNs need access to your subscription, make sure you add them at the specific permission scope and role required for your scenario. For example, sometimes 'Contributor' access at 'Resource Group' scope might be sufficient. In other scenarios you may need 'Reader' access at 'Subscription' scope. Exact permission will vary based on your use case.
+
+- **PowerShell** 
+
+	 ```powershell	 
+        # Below commands will be useful to Read and Remove Password Credentials from Service Principals
```

Comment:
Description seems to be incorrect

Reply:
Done, updated the description to # Below commands will be useful to remove Service Principals